sdk/src/test/java/com/google/cloud/dataflow/sdk/transforms/display/DisplayDataTest.java

```diff
@@ -567,6 +569,45 @@ public void populateDisplayData(Builder builder) {
     // Should not throw
   }
 
+  public void todo() {
+    String[] todo =  {
```

Comment:
@bjchambers: This branch is a work in progress. Feel free to review as I go if you have any design feedback.